src/api/faq/faq.test.js

```diff
@@ -24,62 +24,67 @@ afterAll(async () => {
 });
 
 describe('GET /faq', () => {
-  it('Should respond with a 200 status code', () =>
+  it('Should respond with a 200 status code', done =>
     request(app)
       .get('/faq')
       .expect('Content-Type', /json/)
-      .expect(200));
-  it('Should respond with an array', async () => {
+      .expect(200, done));
+  it('Should respond with an array', async done => {
     const { body } = await request(app)
       .get('/faq')
       .expect(200);
     expect(body).toEqual(expect.any(Array));
+    done();
   });
-  it('Should respond with a non-empty array', async () => {
+  it('Should respond with a non-empty array', async done => {
     const { body } = await request(app)
       .get('/faq')
       .expect(200);
     expect(body).toEqual(expect.any(Array));
     expect(body.length).toBeGreaterThan(0);
+    done();
   });
 });
 
 describe('GET /faq/:id', () => {
-  it('With an invalid id, should respond with an invalid id message', async () => {
+  it('With an invalid id, should respond with an invalid id message', async done => {
     const {
       body: { status, message },
     } = await request(app)
       .get(`/faq/${ids.invalid}`)
       .expect(404);
     expect(status).toBe(404);
```

Comment:
Status code is checked twice. Written like this several times bellow.

Reply:
Fixed